src/Twig/Extension/AceEditorExtension.php

```diff
@@ -65,18 +62,15 @@ public function includeAceEditor(Environment $environment): void
         }
 
         if (!$environment->hasExtension(AssetExtension::class)) {
-            throw new \LogicException('"asset" extension is mandatory if you don\'t include Ace editor by yourself.');
+            throw new \LogicException('"asset" extension is mandatory if you do not include Ace editor by yourself.');
         }
 
-        if (!$this->editorIncluded) {
```

Comment:
`$this->editorIncluded` is already `false` at this point, see https://github.com/norberttech/aceeditor-bundle/pull/70/files#diff-31d475271c3f1d6841658281cc30f41dafbec03880bf340f30ad8e7e572ff183R60